tools/R2nparray/R/R2nparray.R

```diff
@@ -63,7 +63,7 @@ convert.data.frame <- function(X, name='default', strict=TRUE){
 convert.numeric <- function(X, name='default', strict=TRUE) {
     if (length(X) > 1){
         head = paste(name, '= np.array([\n')
-        mid = strwrap(paste(X, collapse=', '), width=76, prefix='    ')
+        mid = strwrap(paste(X, collapse=', '), width=79, indent=3, exdent=4)
```

Comment:
As long as you're tinkering in this file, on line 192 there is an extra space before an "=" that would be soo nice to get rid of.